src/javascript/db.js

```diff
@@ -1,19 +1,29 @@
 import {
-  dbi,
+  firestore,
   songsCollection,
   sheetsCollection,
+  usersCollection,
   functions,
   resultsCollection,
 } from "../helpers/firebaseConfig"; //usersCollection
 import { store } from "../helpers/store";
 
 export function createSong(songInfo) {
-  const { title, artist, image, youtubeId, url, srcMode, tags } = songInfo;
+  const {
+    title,
+    subtitle,
+    artist,
+    image,
+    youtubeId,
+    url,
+    srcMode,
+    tags,
+  } = songInfo;
   let randomId = btoa(parseInt(Date.now() * Math.random())).slice(0, 9);
```

Comment:
If this does not need to be truly random, you can simply do `Math.random().toString(36).substring(0, 9)` assuming you want a 8-alphanumeric-character-long id.

Reply:
I'll take a look

Reply:
Absolutely bamboozling results, don't use.